android/app/build.gradle

```diff
@@ -314,7 +314,7 @@ dependencies {
 if (isNewArchitectureEnabled()) {
     // If new architecture is enabled, we let you build RN from source
     // Otherwise we fallback to a prebuilt .aar bundled in the NPM package.
-    // This will be applied to all the imported transtitive dependency.
```

Comment:
😂 this is fun. such comment was copied directly from the react-native repo

Reply:
Oh, my.... https://cs.github.com/?scopeName=All+repos&scope=&q=%22This+will+be+applied+to+all+the+imported+transtitive+dependency.%22

https://github.com/ReactNativeSchool/react-native-firebase-chat-app/ ?

Reply:
https://github.com/react-native-community/rn-diff-purge/blob/version/0.70.5/RnDiffApp/android/app/build.gradle#L285 Basically this particular phrase appeared in many versions tagged in this repo where people copied from

Reply:
Added to https://github.com/stars/jsoref/lists/to-check-spelling (note that the list mostly grows, and is around 300 items long)...